sql/catalyst/src/main/scala/org/apache/spark/sql/types/ObjectType.scala

```diff
@@ -43,7 +43,7 @@ case class ObjectType(cls: Class[_]) extends DataType {
 
   def asNullable: DataType = this
 
-  override def simpleString: String = cls.getName
+  override def simpleString: String = s"Object[${cls.getName}]"
```

Comment:
this is to make it differentiate between IntegerType and ObjectType with `int` in it. Both show up in the simpleStreaing as `int`.